### DISCUSSION TITLE
How can I change the password when i forgot it ?

### DISCUSSION THREAD

**Discussion by trinv** [ngoduykhanh/PowerDNS-Admin#1626] [Q&A]
### PDA version

0.4.1

### Feature type

New functionality

### Proposed functionality

Hello Team,
I have a question, so How can I change the password when i forgot it ?
Because on the powerdns-admin portal (web browser, i dont know how to change the password)

thanks
Tri Nguyen

### Use case

Hello Team,
I have a question, so How can I change the password when i forgot it ?
Because on the powerdns-admin portal (web browser, i dont know how to change the password)

thanks
Tri Nguyen

**Comment by metrax** [comment]
Before opening an issue, please read the [contribution guide](https://github.com/PowerDNS-Admin/PowerDNS-Admin/blob/master/docs/CONTRIBUTING.md) of the project first.
You request is not an issue, it's a question/support request and should be opened as discussion.

Use an open bcrypt generator like this https://bcrypt-generator.com/ or write a small script to generate a new bcrypt hash for the new password.
Than go to the database and replace the password of the user in the database, table: user.
